services/minespace-web/src/components/Forms/projects/informationRequirementsTable/InformationRequirementsTableForm.tsx

```diff
@@ -70,19 +64,8 @@ export const InformationRequirementsTableForm: FC<InformationRequirementsTableFo
             defaultActiveKey={sideMenuOptions[0]}
             onChange={onTabChange}
             className="vertical-tabs"
-          >
-            {sideMenuOptions.map((tab) => {
-              return (
-                <Tabs.TabPane
-                  tab={formatUrlToUpperCaseString(tab)}
-                  key={tab}
-                  className="vertical-tabs--tabpane"
-                >
-                  {renderTabComponent(mergedRequirements, tabIndex)}
-                </Tabs.TabPane>
-              );
-            })}
-          </Tabs>
+            items={tabItems}
```

Comment:
I was a bit baffled by the buggy UI on this page and did some paring down in the process of figuring out what was wrong. (the flashy IRT page on the bugs channel! Locally, you'll get a ResizeObserver Loop error, possibly depending on the height of your browser but it's the natural height of _my_ browser so it just always happened)

Turned out to be an antd/CSS issue. For the tabs, when there's overflow, antd wants to put a little ellipses button to show the overflow items. But when it's just on the threshold, it's like it doesn't know whether to show it or not, and so the result is constant layout changes. I didn't see a prop in the antd docs I could control this with, so instead I hid the button and made it scroll to show overflow in the scss file. Note that this is the _only instance_ of "vertical-tabs" on MS, so the rule is quite specific to this area. My first idea was to make the menu like the other menus but the page structure was too different for that to be a simple change (updating the UI is going to be a major overhaul FYI)